docs/tasks-overview.md

```diff
@@ -0,0 +1,64 @@
+## Overview of Tasks Provided by kotlinx-benchmark Gradle Plugin
+
+The kotlinx-benchmark plugin creates different Gradle tasks depending on how it is configured.
+For each pair of configuration profile and registered target a task is created to execute that profile on the respective platform.
```

Comment:
It might be worth explicitly stating here that by default a user may run the "benchmark" target and all other things could be used for fine-grained control. It may sound obvious (and it could be comprehended after reading the whole document), but it's always nice to let everyone know that the knowledge described below is not something one has to know to start using the framework.

Reply:
README already contains information about running benchmarks in a basic scenario: https://github.com/Kotlin/kotlinx-benchmark/pull/167/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R395

This documentation is a more comprehensive guide.